examples/RefineMontage2D.cxx

```diff
@@ -29,25 +29,20 @@
 template< typename PixelType, typename AccumulatePixelType >
 void
 montage2D( const itk::TileLayout2D& stageTiles, itk::TileLayout2D& actualTiles,
-           const std::string& inputPath, int peakMethodToUse )
```

Comment:
In #97 I used `itkNotUsed()` just in case they come useful at some point and given that I did not code the examples and did not know the initial intent. Also for `setMontageDirectly`. But if you know better the code, I' fine with the suppression.